projects/plugins/protect/src/js/routes/scan/index.jsx

```diff
@@ -49,16 +69,33 @@ const ScanPage = () => {
 		<OnboardingContext.Provider value={ onboardingSteps }>
 			<AdminPage>
 				<ScanAdminSectionHero />
-				{ ( ! status.error || numThreats ) && (
-					<AdminSection>
-						<Container horizontalSpacing={ 7 } horizontalGap={ 4 }>
-							<Col>
-								<ThreatsList />
-							</Col>
-						</Container>
-					</AdminSection>
-				) }
-				<ScanFooter />
+				<AdminSection>
+					<Container
+						className={ styles[ 'scan-results-container' ] }
+						horizontalSpacing={ 5 }
+						horizontalGap={ 4 }
+					>
+						<Col>
+							<div ref={ setScanResultsAnchor }>
+								<ScanResultsDataView filters={ filters } />
+							</div>
+							{ !! status && ! isScanInProgress( status ) && (
+								<OnboardingPopover
+									id={ hasPlan ? 'paid-scan-results' : 'free-scan-results' }
+									anchor={ scanResultsAnchor }
+									position={ 'top left' }
```

Comment:
On a similar nit note, this placement and position of the arrow on this step could be improved (especially in smaller screen width, it begins to overflow into the admin sidebar menu):
![Screen Shot on 2024-11-26 at 14-59-45](https://github.com/user-attachments/assets/51f21f3e-f6a0-40c2-99b2-9fc36f753d8d)

Reply:
Additionally with the final onboarding step for Severity, ideally it would be closer to field label - understanding that might be challenging or impossible:
![Screen Shot on 2024-11-26 at 15-04-16 (1)](https://github.com/user-attachments/assets/8852bf77-9ca5-4623-822d-bc68c55909b2)